services/src/atdd/src/test/java/org/openkilda/atdd/TopologyEventsBasicTest.java

```diff
@@ -128,14 +147,15 @@ public void a_switch_is_dropped_in_the_middle() throws Exception {
     }
 
     @Then("^all links through the dropped switch will have no health checks$")
-    public void all_links_through_the_dropped_switch_will_have_no_health_checks() throws Exception {
+    public void allLinksThroughTheDroppedSwitchWillHaveNoHealthChecks() throws Exception {
         // Write code here that turns the phrase above into concrete actions
         throw new PendingException();
     }
 
     @Then("^the links disappear from the topology engine\\.$")
-    public void the_links_disappear_from_the_topology_engine() throws Exception {
-        //todo check whether we need to wait until links will disappear or we might delete them instantly when switch goes down
+    public void theLinksDisappearFromTheTopologyEngine() throws Exception {
+        //todo check whether we need to wait until links will disappear or
+        // we might delete them instantly when switch goes down
```

Comment:
Now we know - we must not remove links on switch `down`. It can be control plane outage of switch reboot or some other switch flapping. In any of this events we should not remove ISLs.

Reply:
This is just a checkstyle fix. But I agree that implementation of these test scenarios should align them  with the system design.